### PR TITLE
Fix entry-creator migrations to use dynamic models

### PR DESCRIPTION
**Pull request by gideonthomas**
We referred to these models as they are currently in the code vs. using what is in the db at the time of the migration.

I tested this change and it seems to work fine. I also tested it over production data to make sure that it applies cleanly without any data loss.

### CODE REVIEW

**Comment by alanmoo** [comment]
Talk me through this a bit more. We're changing a migration that already landed on...staging? But has it landed on production? Why are we doing this?

**Comment by gideonthomas** [comment]
Because production breaks when we apply this migration. 
tldr; migrations should always use models as they are in the db at the time of the migration and not as they are in the code.

On staging, since we applied migrations as soon as we landed them, using the models from the code directly wasn't a problem because they would represent what's in the db. On production however, we are applying multiple migrations at a time on the db and the state of the db is not in sync with the models in the code (the migrations are meant to bring them in sync) and in fact, they are very out of sync with the models in the code. So, we can't use the models in the code in a migration and try to apply that migration because the model in the code has properties that are not yet in the db.

**Comment by alanmoo** [comment]
Got it. So to test this I should back up the model to whatever point prod is at and then run all the migrations at once?
